[bug] CSS Styling doesn't work

### What happend?

I get an error when trying to make a style.css Style

<details><summary><code>Error Message</code></summary>

```golang
panic: error marshaling CSS file: line 2: expected style before semicolon

goroutine 1 [running]:
main.main()
        C:/Users/Administrator/GolandProjects/BugReport/main.go:33 +0xa8
exit status 2

```
</details>

### Code example

<details><summary><code>main.go</code></summary>

```golang
package main

import (
	_ "embed"
	g "github.com/AllenDang/giu"
)

//go:embed style.css
var cssStyle []byte

func loop() {
	Window := g.SingleWindow()

	Window.Layout(
		g.PopupModal("Popup Modal").Layout(
			g.Label("This is a popup modal"),
			g.Button("Close").OnClick(func() {
				g.CloseCurrentPopup()
			}),
		),
		g.Row(
			g.Button("Open Popup Modal").OnClick(func() {
				g.OpenPopup("Popup Modal")
			}),
		),
	)
}

func main() {
	wnd := g.NewMasterWindow("Cloud - Universal", 750, 500, g.MasterWindowFlagsNotResizable)

	if err := g.ParseCSSStyleSheet(cssStyle); err != nil {
		panic(err)
	}

	wnd.Run(loop)
}
```
</details>

<details><summary><code>style.css</code></summary>

```css
main {
    background-color: black;
}
```
</details>



### To Reproduce

1. Run my demo
2. will see the crash...


### Version

master

### OS

Windows 11

--- a/StyleIDs_string.go
+++ b/StyleIDs_string.go
@@ -122,7 +122,7 @@ func _() {
 
 const _StyleVarID_name = "alphadisabled-alphawindow-paddingwindow-roundingwindow-border-sizewindow-minValue-sizewindow-title-alignchild-roundingchild-border-sizepopup-roundingpopup-border-sizeframe-paddingframe-roundingframe-border-sizeitem-spacingitem-inner-spacingindent-spacingStyleVarCellPaddingscrollbar-sizescrollbar-roundinggrab-minValue-sizegrab-roundingtab-roundingStyleVarTabBarBorderSizebutton-text-alignselectable-text-alignStyleVarSeparatorTextBorderSizeStyleVarSeparatorTextAlignStyleVarSeparatorTextPaddingStyleVarDockingSeparatorSize"
 
-var _StyleVarID_index = [...]uint16{0, 5, 19, 33, 48, 66, 81, 99, 113, 130, 144, 161, 174, 188, 205, 217, 235, 249, 268, 282, 300, 313, 326, 338, 362, 379, 400, 431, 457, 485, 513}
+var _StyleVarID_index = [...]uint16{0, 5, 19, 33, 48, 66, 86, 104, 118, 135, 149, 166, 179, 193, 210, 222, 240, 254, 273, 287, 305, 323, 336, 348, 372, 389, 410, 441, 467, 495, 523}
 
 func (i StyleVarID) String() string {
 	if i < 0 || i >= StyleVarID(len(_StyleVarID_index)-1) {

